src/utils/nlpUtils/newEntitiesRuler.py

```diff
@@ -0,0 +1,57 @@
+'''
```

Comment:
when merge into devel, we need to add the copyright statement.

Reply:
good point, added